vortex-datetime-dtype/src/temporal.rs

```diff
@@ -23,6 +27,35 @@ pub enum TemporalMetadata {
     Timestamp(TimeUnit, Option<String>),
 }
 
+pub enum TemporalJiff {
+    Time(Time),
+    Date(Date),
+    Timestamp(Timestamp),
+    Zoned(Zoned),
+}
+
+impl Display for TemporalJiff {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            TemporalJiff::Time(x) => write!(f, "{}", x),
+            TemporalJiff::Date(x) => write!(f, "{}", x),
+            TemporalJiff::Timestamp(x) => write!(f, "{}", x),
+            TemporalJiff::Zoned(x) => write!(f, "{}", x),
+        }
+    }
+}
+
+pub enum TemporalJiffError {
```

Comment:
there's a VortexError, we should probably just add JiffError to that enum and then we can transparently convert into this when we return VortexResult

Reply:
Done.